src/net.h

```diff
@@ -347,7 +349,7 @@ class CNode
     bool fInbound;
     bool fNetworkNode;
     bool fSuccessfullyConnected;
-    bool fDisconnect;
+    std::atomic_bool fDisconnect;
```

Comment:
will need to add `#include <atomic>` in the header includes section to get this passing CMake linux

```diff
Index: src/net.h
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/net.h	(revision c452676b3d238432bb93845d856ed697903519d3)
+++ src/net.h	(date 1596265524199)
@@ -20,6 +20,7 @@
 #include "uint256.h"
 #include "utilstrencodings.h"
 
+#include <atomic>
 #include <deque>
 #include <stdint.h>
 
```

Reply:
done